redis/Cargo.toml

```diff
@@ -99,7 +99,7 @@ json = ["serde", "serde/derive", "serde_json"]
 cluster = ["crc16", "rand"]
 script = ["sha1_smol"]
 tls-native-tls = ["native-tls"]
-tls-rustls = ["rustls", "rustls/ring", "rustls-native-certs", "rustls-pemfile", "rustls-pki-types"]
+tls-rustls = ["rustls/std", "rustls/ring", "rustls-native-certs", "rustls-pemfile", "rustls-pki-types"]
```

Comment:
Why is this required?

Reply:
removed

Reply:
I was referring to line 102. I still see the change.

Reply:
Sorry, I was wrong. This is needed in order to build the crate with only the `tls-rustls` flag. You can try it locally.